fili-generic-example/README.md

```diff
@@ -90,4 +95,6 @@ Here are some sample queries that you can run to verify your server:
 
 1. In IntelliJ, go to `File -> Open`
 2. Select the `pom.xml` file at the root of the project
-3. Run `GenericMain` which can be found in `fili-generic-example` (e.g. right click and choose run)
+3. Under `src/main/resources/applicationConfig.properties`, change `bard__non_ui_druid_broker`, 
+`bard__ui_druid_broker`, `bard__druid_coord`, and other properties as needed.
```

Comment:
Yep these are set correctly for running it locally and they work with the wikipedia walkthrough. 

I added the following note in `README.md` to clarify:

> **NOTE:** if you're running this locally and haven't changed any settings (like the Wikipedia example) you can **skip step 3**.